crates/paralegal-policy/src/flows_to.rs

```diff
@@ -70,10 +70,7 @@ pub struct CtrlFlowsTo {
     /// query this representation of the relation.
     pub data_flows_to: IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
 
-    /// The transitive closure of the [`Ctrl::data_flow`] and [`Ctrl::ctrl_flow`] relations representing mixed flows of data and control.
-    /// If a source data-flows to a [`DataSink::Argument`], it flows into its CallSite.
-    /// If a source control-flows into a CallSite, it also flows into all of the [`DataSink::Argument`]s related to it.
-    pub flows_to: IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
+    ctrl_flows_to: IndexMatrix<DataSourceIndex, CallSiteOrDataSink>,
```

Comment:
Do we actually need this? Correct me if I'm wrong but I seem to remember the control flow captured in the SPDG is already transitive. Can't we just use a lookup into the SPDG then?